src/server/game/Entities/Unit/UnitDefines.h

```diff
@@ -269,25 +269,40 @@ enum MovementFlags : uint32
 
 enum MovementFlags2 : uint32
 {
-    MOVEMENTFLAG2_NONE                     = 0x00000000,
-    MOVEMENTFLAG2_NO_STRAFE                = 0x00000001,
-    MOVEMENTFLAG2_NO_JUMPING               = 0x00000002,
-    MOVEMENTFLAG2_UNK3                     = 0x00000004,        // Overrides various clientside checks
-    MOVEMENTFLAG2_FULL_SPEED_TURNING       = 0x00000008,
-    MOVEMENTFLAG2_FULL_SPEED_PITCHING      = 0x00000010,
-    MOVEMENTFLAG2_ALWAYS_ALLOW_PITCHING    = 0x00000020,
-    MOVEMENTFLAG2_UNK7                     = 0x00000040,
-    MOVEMENTFLAG2_UNK8                     = 0x00000080,
-    MOVEMENTFLAG2_UNK9                     = 0x00000100,
-    MOVEMENTFLAG2_UNK10                    = 0x00000200,
-    MOVEMENTFLAG2_INTERPOLATED_MOVEMENT    = 0x00000400,
-    MOVEMENTFLAG2_INTERPOLATED_TURNING     = 0x00000800,
-    MOVEMENTFLAG2_INTERPOLATED_PITCHING    = 0x00001000,
-    MOVEMENTFLAG2_UNK14                    = 0x00002000,
-    MOVEMENTFLAG2_UNK15                    = 0x00004000,
-    MOVEMENTFLAG2_UNK16                    = 0x00008000
+    MOVEMENTFLAG2_NONE                                  = 0x00000000,
+    MOVEMENTFLAG2_NO_STRAFE                             = 0x00000001,
+    MOVEMENTFLAG2_NO_JUMPING                            = 0x00000002,
+    MOVEMENTFLAG2_UNK3                                  = 0x00000004,        // Overrides various clientside checks
+    MOVEMENTFLAG2_FULL_SPEED_TURNING                    = 0x00000008,
+    MOVEMENTFLAG2_FULL_SPEED_PITCHING                   = 0x00000010,
+    MOVEMENTFLAG2_ALWAYS_ALLOW_PITCHING                 = 0x00000020,
+    MOVEMENTFLAG2_UNK7                                  = 0x00000040,
+    MOVEMENTFLAG2_UNK8                                  = 0x00000080,
+    MOVEMENTFLAG2_UNK9                                  = 0x00000100,
+    MOVEMENTFLAG2_UNK10                                 = 0x00000200,
+    MOVEMENTFLAG2_INTERPOLATED_MOVEMENT                 = 0x00000400,
+    MOVEMENTFLAG2_INTERPOLATED_TURNING                  = 0x00000800,
+    MOVEMENTFLAG2_INTERPOLATED_PITCHING                 = 0x00001000,
+    MOVEMENTFLAG2_UNK14                                 = 0x00002000,
+    MOVEMENTFLAG2_CAN_TRANSITION_BETWEEN_SWIM_AND_FLY   = 0x00004000,
+    MOVEMENTFLAG2_UNK16                                 = 0x00008000
 };
 
+enum UnitMoveType
+{
+    MOVE_WALK = 0,
```

Comment:
tabs pls

Reply:
TC codestyle recommends against Tabs.

Reply:
yes and that's no problem when one properly sets up his visual studio to use  tabs4spaces which is what I mean. 
And hi trokli, the stalker of everyone, the one that notices everything and everyone.

Reply:
He didnt mean tabs literally but aligning the = to the same column

Reply:
> He didnt mean tabs literally but aligning the = to the same column

Thanks, that interpretation went over my head. I approve.

Reply:
done